# specify robot track width in meters
track_width: 0.9925
# specify reduction ratio (70:1), if there is no put 1
reduction_ratio: 70
# specify reduction maximum linear velocity (m/s)
# max_vel: 0.95  # 60 m/min @3500rpm
max_vel_x: 0.8  # reduced speed for testing
max_vel_ang: 1.6 
# specify robot wheel circumference (perimeter) in meters 
wheel_circumference: 1.13914122 # D:37cm
encoder_cpr: 6
encoder_min: -65536
encoder_max: 65536 
rate: 30 
odom_topic: odom_noimu
tf_header_frame: aristos_odom
tf_child_frame: aristos_base_footprint